packages/site/src/app/layout.tsx

```diff
@@ -0,0 +1,16 @@
+import Head from 'next/head';
```

Comment:
I think there is a metadata API now

https://nextjs.org/blog/next-13-3#file-based-metadata-api

Reply:
I've not looked at Search yet but GTK